src/main/resources/application.properties

```diff
@@ -0,0 +1,11 @@
+## Spring DATASOURCE (DataSourceAutoConfiguration & DataSourceProperties)
+spring.datasource.url =jdbc:postgresql://localhost:5432/forty_winks_dev
+spring.datasource.username =student
+spring.datasource.password =''
+# Disable feature detection by this undocumented parameter. Check the org.hibernate.engine.jdbc.internal.JdbcServiceImpl.configure method for more details.
```

Comment:
Probably should move auth0 audience and issuer into environment variables

Reply:
Addition of environment variables to store Db details messes up the tests as it says that an application context is missing. So this is a bit more involved. So will look at it later.

Reply:
Ya is possible to do but actually these variables are not sensitive as just refer to local db instances - heroku has its own so not live or staging